### ISSUE TITLE
Filter through Get Route

### ISSUE THREAD

**Issue by skbarbee** [sei-ec-remote/project-4-issues#202]
###### (ex: MERN(mongoose + react), DR(django + react), PEN, etc.)
DR

### What's the problem you're trying to solve?
I am trying to get a Favorite's Index by filtering through the favorites = Favorites.objects.filter(owner=request.user.id) like in the boiler plate. 
### Post any code you think might be relevant (one fenced block per file)
```
class FavoritesView(generics.ListCreateAPIView):
    permission_classes=()
    serializer_class = FavoritesSerializer
    def get(self, request):
        """Index request"""
        # Get all the favorites:
        # favorites = Favorites.objects.all()
        # Filter the favorites by owner, so you can only see your owned favorites
        favorites = Favorites.objects.filter(owner=request.user.id)
        print('this is the request.user', request.user)
        # Run the data through the serializer
        data = FavoritesSerializer(favorites, many=True).data
        return Response({ 'favorites': data })

export const favoritesIndex = (user) => {
	return axios({
		method: 'GET',
		url: apiUrl + '/favorites/',
		
	})
}
```

### If you see an error message, post it here. If you don't, what unexpected behavior are you seeing?
There's no error message until I mess with it 

### What is your best guess as to the source of the problem?
When I print the user it says AnonymousUser 

### What things have you already tried to solve the problem?
I tried sending the User.id as data in the axios request but it just gave me an empty {}
Basically tried manipulating the request anyway I could to include user.id (because it does have the user id in the front end, I can console.log it through out each step) but its usually breaks or doesn't change the behavior of giving me none

**Paste a link to your repository here**
https://github.com/skbarbee/django-comics
